### ISSUE TITLE
Remove Shiny imports from PEcAn.DB::get_workflow_ids

### ISSUE THREAD

**Issue by infotroph** [PecanProject/pecan#1808]
## Description

```
> devtools::check("base/db")
[...]
checking dependencies in R code ... WARNING
'::' or ':::' imports not declared from:
  ‘ncdf4’ ‘shiny’
[...]
```

I submit that DB (and all of the base PEcAn packages) should probably not be importing Shiny. Looks to me as if this comes from just [one line of query.dplyr.R](https://github.com/PecanProject/pecan/blob/develop/base/db/R/query.dplyr.R#L134):

```
  query <- isolate(shiny::parseQueryString(session$clientData$url_search))
```
I propose that we remove this, either by using equivalent base functionality or by moving the shiny call back into a shiny-specific package.

**Comment by ashiklom** [comment]
Should be addressed in a488f721a0c852ddd8035f055a582be9491bead3 and ec17cf068fef0c349ac7c97c05b7a0eddf0216b5 (my fork).

**Comment by infotroph** [comment]
The changes look good, but for some reason GitHub isn't showing me a "new PR" button for them -- dunno if that's GitHub weirdness or something you can fix, but once there's an approve button I'll click it :)

**Comment by ashiklom** [comment]
Closed by #1811.
